src/group/interfaces/group.interface.ts

```diff
@@ -0,0 +1,8 @@
+export interface GroupInterface {
```

Comment:
Please use this schema for the group interface & DTO - https://dev.shikshaplatform.io/swagger/#/Group/GroupController_create